### DISCUSSION TITLE
Asset Inventory: Capturing perils to which each asset is exposed - Capture links to other assets - in event of a hazard - failure chains - Captutre Links to other assets #49

### DISCUSSION THREAD

**Discussion by sreyac** [CriticalAssetManagement/CAMS#125] [General]
Requires some discussion and clarification

**Comment by sreyac** [comment]
Asset would be tagged as vulnerable to a particular hazard

**Comment by luke-feeney** [comment]
multi hop chains of assets noting all the points of failure in a chain

@GavinMendelGleason @KittyJose to look at a multi-hop query and how to visualize 

selection would be 'failure chains this asset is involved in' -> select box that gives failure chains
